docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/compose/ComposeBuilder.java

```diff
@@ -97,4 +113,15 @@ private String getDefaultNetworkName() {
                 .getName() + NETWORK_NAME_SUFFIX;
     }
 
+    private void setNetworkOptions(Object networkOption, CubeContainer cubeContainer){
+        if (networkOption != null) {
+            Map<String, Object> options = (LinkedHashMap) networkOption;
```

Comment:
Why `Object networkOption` as parameter? Don't we immediately assume that this object is a map? We should be more specific about what are the expected inbound and outbound types, otherwise code becomes tricky to follow and reason about